port/win/port_win.h

```diff
@@ -241,6 +241,17 @@ extern void InitOnce(OnceType* once, void (*initializer)());
 #define CACHE_LINE_SIZE 64U
 #endif
 
+// To reduce false cache sharing by enforcing alignment
+#if defined(__clang__) || defined(__GNUC__)
+# define CACHE_ALIGNED(size) alignas(size)
```

Comment:
@zhenyungmail @yuslepukhin then we just get rid of port and put alignas(size) in code, and it should work, right?